lib/renderer/base_chart_painter.dart

```diff
@@ -50,6 +51,7 @@ abstract class BaseChartPainter extends CustomPainter {
     required this.scaleX,
     required this.scrollX,
     required this.isLongPress,
+    required this.isOnTap,
```

Comment:
Maybe some people don’t need this feature.
Can it be null?

Reply:
sure, I can modify

Reply:
I Add a switch to click to show info data, If someone doesn’t need it, can turn it off.
Default is off